### PR TITLE
Provide hosts a different meeting URL (#265, #266)

### LINKED ISSUE #265: If user is host of a Zoom meeting, Join Meeting as Host link should be the host link
There are essentially 2 main types of links for Zoom meetings: host link and participant link. The difference is the letter that shows after the base URL, before the meeting ID. s for hosts, j for participants. Currently we are using the participant link everywhere. We want to switch to using the host link if the current user is the host of the meeting (i.e. if button is Join Meeting as Host)

If they click it and they are already logged in to Zoom, they start the meeting. If they click it and they are not logged in to Zoom, they see:
image

If they click Sign In to Start, they are taken to weblogin. If they sign in as a user who is NOT the host, they get this:
image

### PR DESCRIPTION
**Pull request by jlost**
Closes #265, #266

### CODE REVIEW

**Comment by jlost** [comment]
I targeted this to `zoom-auth-edge-cases` since I wanted to develop with those fixes in place, but will merge `zoom-auth-edge-cases` to `master` first and then change this one's target to `master`.

**Review by ssciolla — APPROVED** [review]
Looks good to me. I left a couple comments. I was able to test the URL uses `j` on the attendee side and `s` on the host side when I am both attendee and host. Again, I couldn't test multiple host scenarios because I would need another fully functional account locally. Interestingly, when I use the `j` link, Zoom still seems to know that I am the host, probably because I'm the owner?

File: src/assets/src/components/meetingTables.tsx [hunk 1]
```diff
@@ -122,8 +122,11 @@ function UnstartedMeetingEditor (props: UnstartedMeetingEditorProps) {
 function StartedMeetingEditor (props: MeetingEditorProps) {
     const attendee = props.meeting.attendees[0];
     const attendeeString = `${attendee.first_name} ${attendee.last_name}`;
-    const joinUrl = props.meeting.backend_metadata?.meeting_url;
-    const role = props.user.id === props.meeting.assignee!.id ? 'Host' : 'Guest';
+    const isHost = props.user.id === props.meeting.assignee!.id;
+    const joinUrl = isHost
+        ? props.meeting.backend_metadata?.host_meeting_url || props.meeting.backend_metadata?.meeting_url
```
[the remark this answers is not in the document]
Comment (jlost) [reply src/assets/src/components/meetingTables.tsx:127]: I was thinking about the edge case of supporting old pre-1.4 Meetings in the UI. In that case, you might be the host, and the metadata might not have a `host_meeting_url`, so if I only included the left side, `joinUrl` would be `undefined`. Since I included the `||`, it'll be `meeting_url`, which is available on pre-1.4 Meetings.

[the remark this answers is not in the document]
Comment (ssciolla) [reply src/assets/src/components/meetingTables.tsx:127]: Nice thinking!
